Using tables to organize media

asnilles opened the pull request:
Pictures in first column, summary in second. Each has only one row. Will look into hiding the tables' lines.
Images' alternate texts updated to reflect the identity of the person in the photo.
Horizontal rules added to improve visual aesthetics when moving from section to section.

asnilles committed:
Using tables to organize media

Pictures in first column, summary in second. Each has only one row. Will look into hiding the tables' lines.
Images' alternate texts updated to reflect the identity of the person in the photo.
Horizontal rules added to improve visual aesthetics when moving from section to section.

asnilles committed:
Replaced markdown syntax images

HTML image syntax provides modifiable options for size and possibly shape.
Will continue to look into a common variable for size.
 
Also revised/reapplied English syntax in various posts.

asnilles committed:
Fix of "style" block

Completed syntax and added trials for table border elimination.
Better solution would be to insert the following in CSS:
>table
>{
>    border-style:none;
>}

shaoweiD commented:
Andrew,
I'm in a conference now, but I can work on this on weekends. 

Thanks,
Shaowei

> On Mar 5, 2015, at 8:47 AM, Andrew Nilles notifications@github.com wrote:
> 
> Pictures in first column, summary in second. Each has only one row. Will look into hiding the tables' lines.
> Images' alternate texts updated to reflect the identity of the person in the photo.
> Horizontal rules added to improve visual aesthetics when moving from section to section.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/shaoweiD/group.github.io/pull/1
> 
> Commit Summary
> 
> Using tables to organize media
> File Changes
> 
> M _posts/2000-01-02-Group.md (88)
> Patch Links:
> 
> https://github.com/shaoweiD/group.github.io/pull/1.patch
> https://github.com/shaoweiD/group.github.io/pull/1.diff
> —
> Reply to this email directly or view it on GitHub.

asnilles committed:
All the "i"s show up as "I"s

Can't figure out how to change it. The font-family call-out isn't working.

asnilles committed:
Create Form

This should be the basics for an input form. Javascript is not present, but should be. It would be the missing link in making most of the form work. I'm not sure how to proceed with that.
Let me know if we have an easier way of doing this.

asnilles committed:
Insert copy of CSS

Comment at end of html is a copy of the stylesheet used at <http://www.cssdesk.com/>

asnilles committed:
Rename Form to Form.html
